examples/textured-floor/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Please, move this file to `examples/floor/textured.html` (I prefer to have several examples per directory, to avoid having a lot of directories)